package.json

```diff
@@ -17,6 +19,8 @@
   },
   "homepage": "https://github.com/webaverse/codespace#readme",
   "dependencies": {
+    "dialog": "github:mozilla/dialog",
```

Comment:
I think we should be using the `mediasoup` upstream rather than `dialog`? Either way the dialog in `mozilla/dialog` is not compatible with our implementation so we might as well make this `versatica/dialog`.

Reply:
using webaverse/dialog now